Cargo.toml

```diff
@@ -16,8 +19,9 @@ edition = "2018"
 nom = { version = "7", default-features = false }
 chrono = { version = "0.4", default-features = false, optional = true }
 num-traits = { version = "0.2", optional = true }
+serde = { version = "1.0", optional = true }
 
 [features]
-default = ["std"]
+default = ["std", "serde"]
```

Comment:
Can we keep that out of the defaults for now?

Reply:
* [ ] is there a way to make this a merge blocker?

Reply:
What do you mean?

Reply:
so that I have to check some box before I can merge